modules/vebal/root-gauges.prisma.ts

```diff
@@ -0,0 +1,76 @@
+import { Chain, PrismaClient } from '@prisma/client';
+import { Address } from 'viem';
+import { prisma as prismaClient } from '../../prisma/prisma-client';
+import { RootGauge, isValidForVotingList } from './root-gauges.onchain';
+
+export class PrismaRootGauges {
+    constructor(private prisma: PrismaClient = prismaClient) {}
```

Comment:
can just use the exported prisma, no need for `this.prisma` below

Reply:
I used dependency injection so that injecting a in unit tests prima mock is easy: https://github.com/beethovenxfi/beethovenx-backend/blob/4eca19b736c7c69232e822aba9f1b27c073bfc91/modules/vebal/root-gauges.prisma.spec.ts#L22

Directly using the import would make tests more difficult (enforcing less direct and TS safe patterns like jest.mock).